### ISSUE TITLE
Finals exam format

### ISSUE THREAD

**Issue by wanxuanong** [nus-cs2103-AY1920S2/forum#128]
Hi can I check if the finals exam format remains the same as the course website?

Part 1: 100 MCQ questions (1 hour, 20 marks)
Part 2: Essay questions (1 hour, 20 marks)

**Comment by raivatshah** [comment]
I think there was an email saying that it won't be the same as past year (in person) papers in terms of the number of questions and time limit. There will be an announcement with the exact details soon.

**Comment by eugenetyc** [comment]
Yup, we received an email stating it's on the 25th April, 1pm. So far it's mostly MCQ and a few short answer questions.
